tests/models/test_model_input_examples.py

```diff
@@ -55,40 +65,53 @@ def test_input_examples(pandas_df_with_all_types):
             .all()
         )
 
+    # NB: Choose nparrays columns that can be encoded using proto_json_utils.pyNumpyEncoder
+    new_df = pandas_df_with_all_types[["integer", "long", "float", "double"]]
```

Comment:
saving tensor example does not work with other types? I guess we do not expect the pandas extension types to work, but what about boolean, string and binary?

Reply:
Other than binary, they work, since binary is converted to base64 encoded strings.